doc/api/child_process.md

```diff
@@ -91,11 +91,11 @@ const spawn = require('child_process').spawn;
 const bat = spawn('cmd.exe', ['/c', 'my.bat']);
 
 bat.stdout.on('data', (data) => {
-  console.log(data);
+  console.log(`${data}`);
```

Comment:
I was following [this line](https://github.com/nodejs/node/blame/master/doc/api/child_process.md#L386). Should I change it as well?